_python_workout_code/7.5.py

```diff
@@ -0,0 +1,6 @@
+def  flip_dict(dict):
+    _dict = {dict[key]:key for key in dict}
```

Comment:
to iterate over the dict keys AND values, you should use `.items()` : https://docs.python.org/3/tutorial/datastructures.html#looping-techniques

so it should be as simple as 
```
def flip_dict(d):
    return {v: k for k, v in d.items()}
```